index.html

```diff
@@ -64,207 +71,244 @@ <h2 class="media-h2">LET'S CONNECT</h2>
+                                        <li class="role">2015</li>
+                                    </ul>
+                                </div>
                             </div>
+                            
                             <div>
-                                <ul class="roles-tonic1">
-                                    <li class="role-tonic1">Back End Dev</li>
-                                    <li class="role-tonic1">2015</li>
+                                <p class="p-deskribe">
+                                    A daily selection of privately personalized reads; no accounts or  sign-ups required.
+                                </p>
+                            </div>
+                            
+                            <div>                                
+                                <ul class="tags-ul">
+                                    <li class="tag"><a href="#">html</a></li>
+                                    <li class="tag"><a href="#">css</a></li>
+                                    <li class="tag"><a href="#">javaScript</a></li>
                                 </ul>
                             </div>
-                        </div>
-
-                        <div >
-                            <p class="p-tonic1">
-                                A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.
-                            </p>
-                        </div>
-
-                        <div>
-                            <ul class="tags-ul-tonic1">
-                                <li class="tag-li-tonic1"><a href="#">html</a></li>
-                                <li class="tag-li-tonic1"><a href="#">css</a></li>
-                                <li class="tag-li-tonic1"><a href="#">javaScript</a></li>
-                            </ul> 
                             <div>
                                 <button class="btn btn-enabled-tonic1">See Project</button>
                             </div>
+
                         </div>
+                        
                     </div>
                 </div>
             </div>
-            
 
-            <div class="item-tonic1">
-                <div class="card-tonic1">
-                    <div class="card-body-tonic1">
-                        <img class="card-image-tonic1" src="./images/works2.png" alt="works1">
-                        <h3 class="card-title-tonic1">Multi-Post <br> Stories</h3>
-                        <div class="canopy-frm-tonic1">
-                            <div class="client-tonic1">
-                                <h3>CANOPY</h3>
+            <div class="item">
+                <div class="card">
+                    <div class="card-body">
+                        <div class="card-image img-right">
+                            <img class="card-image" src="./images/desk-works2.png" alt="works1">
```

Comment:
- [ ] When making a website responsive, it's very important that we do not break the previous layouts we created, unfortunately, the layout is breaking here, this class used to be called on your CSS: card-image-tonic1 and was replaced by card-image1, kindly fix the class here so the properties for width and height apply again

As this and other images mentioned below without the class are breaking the site on mobile:
![image](https://user-images.githubusercontent.com/73140461/144422215-ebe96b63-5d1d-4c51-8858-ddc540de8d34.png)

Reply:
corrected, 
thanks for this comment
very useful.